workspaces/kubernetes-native-dxp/k8s/remoteappa/ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: remote-app-a-localhost
```

Comment:
see comment about not having a ingress per service...

Reply:
fixed.